query.go

```diff
@@ -11,9 +11,38 @@ import "C"
 
 import "unsafe"
 
+const (
+	// notmuch_query_search_messages and notmuch_query_search_threads
+	// will return all matching messages/threads regardless of exclude status.
+	// The exclude flag will be set for any excluded message that is
+	// returned by notmuch_query_search_messages, and the thread counts
+	// for threads returned by notmuch_query_search_threads will be the
+	// number of non-excluded messages/matches.
+	QUERY_EXCLUDE_FLAG = C.NOTMUCH_EXCLUDE_FLAG
```

Comment:
Let's omit the `QUERY_*` prefix, and just call it `EXCLUDE_FLAG` (and likewise with the other constants).